chia/server/node_discovery.py

```diff
@@ -64,7 +64,8 @@ def __init__(
         self.cleanup_task: Optional[asyncio.Task] = None
         self.initial_wait: int = 0
         self.resolver = dns.asyncresolver.Resolver()
-        self.pending_outbound_connections: Set = set()
+        self.pending_outbound_connections: Set[str] = set()
```

Comment:
I don't understand why this is just host names rather than `(host, port)`.

Reply:
everyone must use the same port. partly because we've kind of decided that, but also because DNS doesn't come with ports